pkg/referrerstore/oras/utils.go

```diff
@@ -53,3 +53,13 @@ func OciDescriptorToReferenceDescriptor(ociDescriptor oci.Descriptor) ocispecs.R
 		ArtifactType: ociDescriptor.ArtifactType,
 	}
 }
+
+func OciManifestToReferenceManifest(ociManifest oci.Manifest) ocispecs.ReferenceManifest {
```

Comment:
suggestion to add unit test if time allows

Reply:
added test